### DISCUSSION TITLE
Different appearance between version 3.1.3 and 4.0

### DISCUSSION THREAD

**Discussion by konstantinos-iatridis** [coin3d/coin#530] [Q&A]
Hello,
I switched from version coin3d 3.1.3 to coin3d 4.0 and I realized that the color of shapes in my application appear darker.
Is it my impression? If it is actually the case is a way to revert to the appearance of 3.1.3 while using 4.0 since I don't want to loose the Unicode support of 4.0.
Thank you in advance for an answer,
Konstantinos Iatridis

**Comment by VolkerEnderlein** [comment]
Hi Konstantinos,

I am not aware that the rendering especially the color handling in Coin changed between 3.1.3 and 4.0.0. Can you add some more information about what you draw, what features you use (textures...) on wich platform you are seeing the issue, what GUI toolkit you use, ...? Maybe you could add some render images done with coin3.1.3 and coin4.0.0 to see the differences. It could be related to lighting and need to be fixed.

Cheers, Volker
